### ISSUE TITLE
Invalid cchar value while processing juliadoc.vim

### ISSUE THREAD

**Issue by StevenWhitaker** [JuliaEditorSupport/julia-vim#303]
When I try to open the documentation for a function (with `K`), I get the following error:
```
Error detected while processing /home/steven/.local/share/nvim/site/pack/packer/start/julia-vim/syntax/juliadoc.vim:
line   43:
E844: invalid cchar value
Press ENTER or type command to continue
Error detected while processing /home/steven/.local/share/nvim/site/pack/packer/start/julia-vim/syntax/juliadoc.vim:
line   43:
E475: Invalid argument: juliadocCodeLatex "\\\\nolinebreak" conceal cchar=<2060> display contained
Press ENTER or type command to continue
```
After pressing enter, the documentation opens up and seems to work correctly. So it's not a huge deal, just slightly annoying.

I am using nvim v0.7.2 on Manjaro Linux. Let me know if you need more information.

**Comment by machakann** [comment]
I believe the problem would be fixed by #305.

**Comment by carlobaldassi** [comment]
This should be fixed now.
